uploadNeuroDB/NeuroDB/objectBroker/ConfigOB.pm

```diff
@@ -467,6 +473,90 @@ sub getComputeSnrModalities {
 }
 
 
+=head3 getExcludedSeriesDescription()
+
+Get the excluded_series_description Config setting
+
+RETURN: an array (possibly empty) of the series description to exclude from import
+
+=cut
+sub getExcludedSeriesDescription {
+    my $self = shift;
+
+    return &$getConfigSettingRef($self, EXCLUDED_SERIES_DESCRIPTION);
+}
+
+
+=head3 getModalitiesToDeface()
+
+Get the modalities_to_deface Config setting
+
+RETURN: an array (possibly empty) of the modalities to run the defacing pipeline on
+
+=cut
+sub getModalitiesToDeface {
+    my $self = shift;
+
+    return &$getConfigSettingRef($self, MODALITIES_TO_DEFACE);
+}
+
+
+=head3 getBidsDatasetAuthors()
+
+Get the bids_dataset_authors Config setting
+
+RETURN: an array (possibly empty) of the authors to use for a BIDS dataset
+
+=cut
+sub getBidsDatasetAuthors {
+    my $self = shift;
+
+    return &$getConfigSettingRef($self, BIDS_DATASET_AUTHORS);
+}
+
+
+=head3 getBidsAcknowledgmentsText()
+
+Get the bids_acknowledgments_text Config setting
+
+RETURN: an array (possibly empty) of the acknowledgment text to use for a BIDS dataset
```

Comment:
I don't think this method returns an array but a string. Also, when you do not define an acknowledgment text, this method returns `undef` (not the empty string `''`). Is this the intended behavior?